plugins/obs-outputs/CMakeLists.txt

```diff
@@ -6,6 +6,10 @@ if(NOT DEFINED ENABLE_RTMPS)
       CACHE STRING "Enable RTMPS support with mbedTLS" FORCE)
   set_property(CACHE ENABLE_RTMPS PROPERTY STRINGS AUTO ON OFF)
 endif()
+if(NOT ENABLE_RTMPS)
+  obs_status(DISABLED "RTMPS support with mbedTLS")
+  return()
```

Comment:
This `return()` shall be removed.

Reply:
It would also make more sense to add this output after line 109 where crypto is being disabled.